package.json

```diff
@@ -24,20 +24,20 @@
     "start": "electron .",
     "clean": "rm -rf release doc/Sia-UI node_modules Sia config.json **/*.swp npm-debug.log",
     "fresh": "npm run clean && npm install && npm start",
-    "test": "electron-mocha",
-    "doc": "jsdoc -c .jsdocrc",
+    "test": "npm run lint --silent && electron-mocha",
     "lint": "jshint . --verbose --exclude-path=.jshintignore",
+    "doc": "jsdoc -c .jsdocrc",
     "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
     "sia-import": "(test -d Sia || mkdir -p Sia) && (cp $GOPATH/bin/siad Sia & cp $GOPATH/bin/siac Sia)",
     "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
     "sia-reimport": "npm run sia-repo && npm run sia-kill && npm run sia-import",
     "release-args": "electron-packager . Sia-UI --app-version=0.4.8-beta --version=0.34.1 --overwrite --prune --ignore=release --out=release/v0.4.8-beta --icon=app/assets/icon",
     "release": "npm run sia-import ; npm run release-args -- --all && ./.finalizeRelease.sh",
-    "release-lin64": "npm run release-args -- --platform=linux --arch=x64",
-    "release-lin32": "npm run release-args -- --platform=linux --arch=ia32",
-    "release-win64": "npm run release-args -- --platform=win32 --arch=x64",
-    "release-win32": "npm run release-args -- --platform=win32 --arch=ia32",
-    "release-mac": "npm run release-args -- --platform=darwin --arch=x64"
+    "release-lin64": "npm run release-args -- --platform=linux --arch=x64 && ./.finalizeRelease.sh",
+    "release-lin32": "npm run release-args -- --platform=linux --arch=ia32 && ./.finalizeRelease.sh",
+    "release-win64": "npm run release-args -- --platform=win32 --arch=x64 && ./.finalizeRelease.sh",
+    "release-win32": "npm run release-args -- --platform=win32 --arch=ia32 && ./.finalizeRelease.sh",
+    "release-mac": "npm run release-args -- --platform=darwin --arch=x64 && ./.finalizeRelease.sh"
```

Comment:
takes out `default_app` and zips up the release for each of the individual release commands.